tree/tree/inc/TVirtualIndex.h

```diff
@@ -43,6 +43,7 @@ class TVirtualIndex : public TNamed {
    virtual const char    *GetMinorName()    const = 0;
    virtual TTreeFormula  *GetMajorFormulaParent(const TTree *parent) = 0;
    virtual TTreeFormula  *GetMinorFormulaParent(const TTree *parent) = 0;
```

Comment:
you can remove `GetMajorFormulaParent` and `GetMinorFormulaParent` from here. I only introduced them very recently and only for the piece of code that you modify later

Reply:
I sync my commits with your backport :)
I also remove `virtual` specifier for these special methods